modules/configuration/js/project.js

```diff
@@ -18,9 +18,15 @@ $(document).ready(function() {
         }
         
         var errorClosure = function(i, form) {
-            return function() {
-                $(form.find(".saveStatus")).text("Failed to save, same name already exist!").css({ 'color': 'red'}).fadeIn(500).delay(1000).fadeOut(500);
+          if (isNaN(recruitmentTarget)) {
```

Comment:
Can you create an issue (not for 21.0) saying we should have an integer or numeric config type? I think it makes more sense than hardcoding special cases for some fields.

Reply:
https://github.com/aces/Loris/issues/4833